packages/store/src/ivy/ensure-state-class-is-injectable.ts

```diff
@@ -1,19 +1,21 @@
 import { StateClass } from '@ngxs/store/internals';
 
-import { ivyEnabledInJitMode } from './ivy-enabled-in-jit-mode';
+import { ivyEnabledInDevMode$ } from './ivy-enabled-in-dev-mode';
```

Comment:
I don't think that the `$` on the end is necessary. That is an observable convention.

Reply:
OK. Tslint whines on the shadowed variable, what would you propose?